pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -312,40 +316,77 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	return plugin.deleteFromNetwork(plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
-func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
+func PodDesc(namespace, name string, id kubecontainer.ContainerID) string {
 	return fmt.Sprintf("%s_%s/%s", namespace, name, id.ID)
 }
 
+func GetIpFromResult(res *cnicurrent.Result, rt *libcni.RuntimeConf) string {
+	klog.V(4).Infof("res: %s", res.String())
+	rt_json, err := json.Marshal(rt)
```

Comment:
I would make these log messages much more descriptive. And instead of %s (for Result) or marshalling to JSON (for RuntimeConf) why not just log them with %v so the Go object gets printed directly?